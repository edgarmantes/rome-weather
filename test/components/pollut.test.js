import React from 'react'
import renderer from 'react-test-renderer';
import Pollut from '../../src/components/pollut';

describe('<Pollut />' , () => {


	let wrapper

	beforeEach( ()=>{
		wrapper = shallow(<Pollut co={"200"} />)
	})

	it('Renders Pollut component properly', () => {
		expect(wrapper.length).toEqual(1)
	});

	it('Adds Pollut passed in value to this.props.day', () => {
		expect(wrapper.find('p').text()).toEqual("200ppm")
	})

	it('Snapshot of Pollut', () => {
        const renderedValue =  renderer.create(<Pollut />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });
});
