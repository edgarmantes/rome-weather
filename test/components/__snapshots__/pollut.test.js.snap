// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pollut /> Snapshot of Pollut 1`] = `
<div
  className="pollut-container"
>
  <h3>
    Carbon Monoxide at sea level:
  </h3>
  <p>
    ppm
  </p>
</div>
`;
